settings/installer/packstack/messaging/qpidd.yml

```diff
@@ -1,6 +1,3 @@
-messaging:
-    variant: qpid
-
-packstack:
-    config:
-        CONFIG_AMQP_BACKEND: "{{ !lookup installer.messaging.variant }}"
+variant: qpid
```

Comment:
so, let's have all these CONFIG_AMQP_BACKEND="{{ !lookup installer.messaging.variant }}" preset in settings/packstack.yml  and add `--messaging-variant` argument with choices `[qpid, none, rabbitmq]` and default